### PR TITLE
Update developer guide

### PR DESCRIPTION
**Pull request by turtle96**

### CODE REVIEW

**Comment by turtle96** [comment]
@CS2103AUG2016-W15-C3/documentation 
Please review

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -540,46 +540,45 @@ NIL
...
 **Pros:**<br>
 1. Has support for cross-platform operations<br>
-2. Can Share tasks with other people and manage them<br> 
+2. Can share tasks with other people and manage them<br> 
 3. Can categorize tasks into different categories<br>
 4. Can attach different types of file inside the task, such as photos, PDF and PowerPoint<br>
 
 **Cons:**<br>
-1. Has limited priority level<br>
+1. Has limited number ofpriority levels<br>
```
Comment (Mamtha3005) [inline docs/DeveloperGuide.md]: space between of and priority

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -540,46 +540,45 @@ NIL
...
 2. Has no undo operation<br>
 
 #### Remember the Milk
 **Pros:**<br>
-1. Can Use offline and sync once internet connection is established<br>
-2. Can handle some natural language processing by saving deadlines from task information itself (e.g. Do math homework by tomorrow: Saves task with deadline set to startDate of tomorrow)<br>
-3. Can Undo operation when marking tasks as done (recover from accidentally clicking done)<br>
-4. Can set priority with some form of order<br>
+1. Can be used offline and syncs once internet connection is established<br>
```
Comment (Mamtha3005) [inline docs/DeveloperGuide.md]: be used and syncs is not consistent... need to change

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -540,46 +540,45 @@ NIL
...
+2. Can handle some natural language processing by saving deadlines from task information itself (e.g. Do math homework tomorrow: Saves task with deadline set to tomorrow)<br>
+3. Can undo when marking tasks as done (recover from accidentally marking a task as done)<br>
+4. Can set priority, and will list tasks by priority<br>
```
Comment (Mamtha3005) [inline docs/DeveloperGuide.md]: set priority and list tasks

**Comment by Mamtha3005** [comment]
Documentation Reviewed

**Comment by Mamtha3005** [comment]
Documentation Reviewed

**Pull request merged**